docs/UserGuide.md

```diff
@@ -227,11 +227,11 @@ Students’ data are saved in the hard disk automatically after any command that
 Action | Format, Examples
 --------|------------------
 **Help** | `help`
-**Add** | `add n/NAME m/MATRICULATION_NUMBER e/NUS_EMAIL_ADDRESS [t/TAG]…​` <br> e.g., `add n/Rainer Lam m/A0123456C e/rainerlam@u.nus.edu t/smart`
-**List** | `list`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find Justin Bieber`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX n/UPDATED_NAME`<br> e.g., `edit 2 n/John Cena`
-**Clear** | `clear`
+**Add** | `addstu n/NAME m/MATRICULATION_NUMBER e/NUS_EMAIL_ADDRESS [t/TAG]…​` <br> e.g., `addstu n/Rainer Lam m/A0123456C e/rainerlam@u.nus.edu t/smart`
```

Comment:
Should this be **AddStu** instead of Add? Because there is a AddSes later

Reply:
I believe this is to describe the action that users want to perform. I have updated them to "Add Student, Delete Student and so on". Thank you for pointing it out!